types/secure-password/secure-password-tests.ts

```diff
@@ -0,0 +1,15 @@
+import * as SecurePassword from 'secure-password';
```

Comment:
namespaces are not callable, instead use: 
```suggestion
import SecurePassword = require('secure-password')
```